9.1.Индексация/9.1.11.py

```diff
@@ -0,0 +1,8 @@
+s = input()
+flag = "Цифр нет"
+
+for i in range(len(s)):
+    if s[i] in "0123456789":
```

Comment:
Опять же, вместо обращения к символам по индексу гораздо проще использовать перебор символов:
```suggestion
for с in s:
    if с in "0123456789":
```